com.unity.render-pipelines.high-definition/Runtime/Unity.RenderPipelines.HighDefinition.Runtime.asmdef

```diff
@@ -38,6 +40,16 @@
             "name": "com.unity.modules.nvidia",
             "expression": "1.0.0",
             "define": "ENABLE_NVIDIA_MODULE"
+        },
+        {
+            "name" : "com.unity.mathematics",
+            "expression" : "1.2.1",
+            "define" : "ENABLE_MATHEMATICS_1_2_1"
+        },
+        {
+            "name": "com.unity.burst",
+            "expression": "1.5.0",
+            "define": "ENABLE_BURST_1_0_0_OR_NEWER"
```

Comment:
Is it 1.5.0 or 1.0.0? The expression should probably agree with the define name.

Reply:
This is matching URP's define, also the define is 1_0_0_OR_NEWER, so this is newer, but its on par with what URP has.

Reply:
Ok, could be there's a mistake on URP as well. Still, if the define says "1_0_0_OR_NEWER", then the expression should be "1.0.0".  If the expression is "1.5.0", the define should read "1_5_0".

Reply:
the way I undesrtood it was "_OR_NEWER" covered essentially anything above or equal to 1.0.0. Ill make the change but if I have to touch URP that might have to be done in a separate PR since it could break things / impact things. Ill do this change just on HDRP.

Reply:
Ah I see. Yeah I'm fine with not touching URP right now. My understanding is that when you write `"expression": "1.5.0"`, it's equivalent to saying "enable this define if package version >= 1.5.0", and the define name should have the equivalent name - for this expression that's `..._1_5_0_OR_NEWER".